core/ir/opnd_api.h

```diff
@@ -1284,10 +1284,10 @@ enum {
     DR_REG_LAST_VALID_ENUM = DR_REG_FCSR, /**< Last valid register enum. */
     DR_REG_LAST_ENUM = DR_REG_FCSR,       /**< Last value of register enums. */
 
-    DR_REG_START_64 = DR_REG_X0,  /**< Start of 64-bit general register enum values. */
```

Comment:
Why remove "general"?  That is what distinguishes this from SIMD (x86 has 64-bit SIMD registers: MMX).

Reply:
I thought it is used to distinguish integer regs from float regs. Sorry, reverted in https://github.com/DynamoRIO/dynamorio/pull/6095/commits/4dc5fb5ebcab5db0f70080a3ebd3f4a404457d1f.